docs/MoreAboutPersistentStorage.md

```diff
@@ -37,9 +35,11 @@ Taking into consideration this can result in wasted space, we have allowed setti
     StoreStorageSize: 10 Gi
 
 ```
+
 **Note**: The default storage class, as configured in the system, is used for configuring the persistent volumes automatically unless a different storage class is specified in the custom resource specification. If no storage class exists, for example in default OpenShift bare metal installations, a storage class must be created and specified or the installation of observability fails.
-<br>
-<br>
 
 ## Object Store
-In addition to the persistent volumes previously mentioned, the time series historical data is stored in object stores. Thanos uses object storage as the primary storage for metrics and metadata related to them. Details about the object storage and downsampling will be provided in another document.
\ No newline at end of file
+
+In addition to the persistent volumes previously mentioned, the time series historical data is stored in object stores. Thanos uses object storage as the primary storage for metrics and metadata related to them. Details about the object storage and downsampling are provided in another document.
+
+//what's the link to the "other document?
```

Comment:
@bjoydeep sign-off is good now, I left a comment for you here

Reply:
@dockerymick this document is being worked on. I will add a link after its done.